src/utility/ProtoSerializable.cpp

```
@@ -1,7 +1,74 @@
+    proto::common::ProtoSerializableMessage message;
+    message.set_schema_name(serializeSchema().schemaName);
+    message.set_metadata_only(metadataOnly);
+
+    const auto serializedMessage = serializeProto(metadataOnly);
+    if(!serializedMessage.empty()) {
+        message.set_proto_message(serializedMessage.data(), serializedMessage.size());
+    }
+    writeMsgBinaryFile(resolveDataPath(path), message);
+}
+
+void ProtoSerializable::load(const std::filesystem::path& path) {
```

Cppcheck (2.21.0)

[style] The function 'load' is never used.

(unusedFunction)